com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -352,6 +333,7 @@ internal Vector3 Scale
         /// If using different values, please use RPCs to write to the server. Netcode doesn't support client side network variable writing
         /// </summary>
         // This is public to make sure that users don't depend on this IsClient && IsOwner check in their code. If this logic changes in the future, we can make it invisible here
+        // TODO: With recent updates 08-2022, this should be changed to a private set.
```

Comment:
Is there a ticket or an issue to track this if we ever create a breaking change version?

Reply:
Not yet, I am still drafting (but please do continue to review, poke, prod, and challenge any changes... the more eyes the better IMO).  
This is something I am going to put in a list (will send you a link to the document)